src/Collections-Support/Association.class.st

```diff
@@ -81,6 +74,13 @@ Association >> storeOn: aStream [
 	aStream nextPut: $)
 ]
 
+{ #category : #testing }
+Association >> useFullDefinition [
```

Comment:
One notes Marcus.
Considering other true/false methods it should begin with "uses" instead of use.

But generally I would try to avoid this method at all. Seems it is used only for class definition logic and it can be simple printStringForClassDefinition with friends without introducing that "type" message

Reply:
printStringForClassDefinition would be a good solution of we would not use this in addition to check if a class needs to print a slot definition...

Reply:
maybe "needsFullDefinition" would be better

Reply:
committed #3320 which changes it to #needsFullDefinition